R/data-column.R

```diff
@@ -12,69 +12,64 @@ DataColumn <- R6::R6Class(
   active = list(
     #' @field values Returns the values defined in the column
     values = function(value) {
-      private$wrapVectorProperty("Value", "ValuesAsArray", value, "ValuesAsArray")
+      private$.wrapVectorProperty("Value", "ValuesAsArray", value, "ValuesAsArray")
     },
     #' @field name Returns the name of the column  (Read-Only)
     name = function(value) {
-      private$wrapReadOnlyProperty("Name", value)
+      private$.wrapReadOnlyProperty("Name", value)
     },
     #' @field unit The base unit in which the values are defined (Read-Only)
     unit = function(value) {
-      if (!missing(value)) {
-        value <- enc2utf8(value)
```

Comment:
Enforcing conversion to utf8 is not performed in rSharp:

https://github.com/Open-Systems-Pharmacology/rSharp/blob/2204ca57c5e8e84c733f3301f4ee9407a76c768f/R/utilities-static.R#L91